src/main/kotlin/team/sfe/server/domain/closure/domain/Closure.kt

```diff
@@ -0,0 +1,27 @@
+package team.sfe.server.domain.closure.domain
+
+import jakarta.persistence.Column
+import jakarta.persistence.Embeddable
+import jakarta.persistence.EmbeddedId
+import jakarta.persistence.Entity
+import jakarta.validation.constraints.NotNull
+import java.io.Serializable
+
+@Entity
+class Closure(
+    @EmbeddedId
+    val closurePk: ClosureId,
+
+    @field:NotNull
+    @Column(columnDefinition = "INT")
+    val depth: Int
+)
+
+@Embeddable
+data class ClosureId(
+    @Column(columnDefinition = "INT")
```

Comment:
```suggestion
    @Column(columnDefinition = "INT UNSIGNED")
```
p1; entity에 선언된 INT도 이렇게 변경해주세요